### PR TITLE
Update how-to-install-composer-programmatically.md

### PR DESCRIPTION
**Pull request by sana8237**
wget https://raw.githubusercontent.com/composer/getcomposer.org/1b137f8bf6db3e79a38a5bc45324414a6b1f9df2

### CODE REVIEW

File: doc/faqs/how-to-install-composer-programmatically.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ For example:
 
 ```bash
 wget https://raw.githubusercontent.com/composer/getcomposer.org/1b137f8bf6db3e79a38a5bc45324414a6b1f9df2/web/installer -O - -q | php -- --quiet
-```
+```doc/faqs/how-to-install-composer-programmatically.md
```
Comment (sana8237) [inline doc/faqs/how-to-install-composer-programmatically.md]: Add

**Comment by stof** [comment]
the diff of your PR looks totally wrong.

File: doc/faqs/how-to-install-composer-programmatically.md (hunk 1 above)
Comment (Seldaek) [inline doc/faqs/how-to-install-composer-programmatically.md]: I'm not adding this no.. it makes no sense to add that there.

**Comment by Seldaek** [comment]
Sorry but closing for now, if you can explain then we will reopen.

**Comment by sana8237** [comment]
https://github.com/composer/composer/pull/6422#discussion_r116867322

**Comment by sana8237** [comment]
https://raw.githubusercontent.com/composer/getcomposer.org/1b137f8bf6db3e79a38a5bc45324414a6b1f9df2/web/installer

**Comment by sana8237** [comment]
> @@ -36,7 +36,7 @@ For example:
 
 ```bash
 wget https://raw.githubusercontent.com/composer/getcomposer.org/1b137f8bf6db3e79a38a5bc45324414a6b1f9df2/web/installer -O - -q | php -- --quiet
-```
+```doc/faqs/how-to-install-composer-programmatically.md

**Pull request closed without merging**